modules/installation-network-user-infra.adoc

```diff
@@ -266,31 +296,91 @@ A working configuration for the Ingress router is required for an
 plane initializes.
 ====
 
-ifdef::vsphere[]
-[discrete]
-== Ethernet adaptor hardware address requirements
+[id="installation-load-balancing-user-infra-example_{context}"]
+== Example load balancer configuration for user-provisioned clusters
 
-When provisioning VMs for the cluster, the ethernet interfaces configured for
-each VM must use a MAC address from the VMware Organizationally Unique
-Identifier (OUI) allocation ranges:
+This section provides an example API and application ingress load balancer configuration that meets the load balancing requirements for user-provisioned clusters. The sample is an `/etc/haproxy/haproxy.cfg` configuration for an HAProxy load balancer. The example is not meant to provide advice for choosing one load balancing solution over another.
 
-* `00:05:69:00:00:00` to `00:05:69:FF:FF:FF`
-* `00:0c:29:00:00:00` to `00:0c:29:FF:FF:FF`
-* `00:1c:14:00:00:00` to `00:1c:14:FF:FF:FF`
-* `00:50:56:00:00:00` to `00:50:56:FF:FF:FF`
-
-If a MAC address outside the VMware OUI is used, the cluster installation will
-not succeed.
-endif::vsphere[]
+[NOTE]
+====
+In the example, the same load balancer is used for the Kubernetes API and application ingress traffic. In production scenarios you can deploy the API and application ingress load balancers separately so that you can scale the load balancer infrastructure for each in isolation.
+====
 
-ifdef::vsphere[]
-:!vsphere:
-endif::[]
+.Sample API and application ingress load balancer configuration
+[%collapsible]
+====
+[source,text]
+global
```

Comment:
(Collapsible content suggestions) You can add `----` after `[source,text]` to start the code block.

Reply:
@lbarbeevargas thank you for all of these details, this is very helpful. I have updated the PR with your recommendations so that the call outs are now included within the collapsible blocks in each instance.